module/trace/trace.go

```diff
@@ -132,6 +132,16 @@ func (t *OpenTracer) StartSpanFromParent(
 	return t.Tracer.StartSpan(string(operationName), opts...)
 }
 
+func (t *OpenTracer) WithSpanFromContext(ctx context.Context,
```

Comment:
Could we add a short godoc for this function?

Reply:
fixed in https://github.com/onflow/flow-go/pull/385/commits/e6e6a99e6cbe4da5a7fe6dbdc1cd2accca7893f5.